data_summaries/KIWI.json

```diff
@@ -0,0 +1,54 @@
+{
+    "KIWI": {
+        "Unique Dataset Identifier": "KIWI",
+        "Dataset Name": "KIWI",
+        "Paper Title": "KIWI: A Dataset of Knowledge-Intensive Writing Instructions for Answering Research Questions",
+        "Dataset URL": "https://www.cs.utexas.edu/~fxu/kiwi/",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/fangyuan/kiwi",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2403.03866",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "KIWI",
+        "Collection URL": "https://www.cs.utexas.edu/~fxu/kiwi/",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Question Answering",
+            "Scientific Knowledge Retrieval",
+            "Question Understanding",
+            "Open Question Answering",
+            "Academic Tasks"
+        ],
+        "Text Sources": [
+            "User interactions",
```

Comment:
Will need to add this in here too: https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/main/constants/domain_groups.json#L323